models/app.go

```diff
@@ -95,6 +110,31 @@ func (a *App) ExperimentalServices() []*ExperimentalService {
 	return services
 }
 
+// RestEndpoints returns a list of all REST endpoints in Packages.
+func (a *App) RestEndpoints() []*RestEndpoint {
+	endpoints := make([]*RestEndpoint, 0)
+	for _, pkg := range a.Packages {
+		for _, service := range pkg.Services {
+			for _, method := range service.Methods {
+				if method.RestMapping != nil && method.RestMapping.Path != "" {
```

Comment:
nit: flip the condition and add a `continue` to have the "happy path" one level less indented?

Reply:
Makes sense. Done 👍